.github/workflows/bridge.yml

```diff
@@ -0,0 +1,157 @@
+name: tardis-carsus-compatibility-check
+
+on: 
+  workflow_dispatch:
+    inputs:
+      git-ref:
+        description: Git Ref (Optional)    
+        required: false
+        
+env:
+  XUVTOP: /tmp/chianti
+  CHIANTI_DL_URL: https://download.chiantidatabase.org
+  CHIANTI_DB_VER: CHIANTI_v9.0.1_database.tar.gz
+  NBCONVERT_CMD: jupyter nbconvert --execute --ExecutePreprocessor.timeout=600 --to html
+  CMFGEN_DL_URL: http://kookaburra.phyast.pitt.edu/hillier/cmfgen_files
+  CMFGEN_DB_VER: atomic_data_15nov16.tar.gz
+  # original reference data
+  REF1_PATH: ${{ github.workspace }}/tardis-refdata/unit_test_data_org.h5
+  # generated reference data using the new atomic file 
+  REF2_PATH: ${{ github.workspace }}/tardis-refdata/unit_test_data.h5
+  PYTEST_FLAGS: --tardis-refdata=${{ github.workspace }}/tardis-refdata 
+                --generate-reference
+
+defaults:
+      run:
+        shell: bash -l {0}
+
+jobs:
+  carsus-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: atharva-2001/carsus
```

Comment:
Why are you checking out your own fork (or am I misunderstanding something)?

Reply:
I thought I would have to clone Carsus with my branch to have the workflow, but I was wrong. This has been fixed now(https://github.com/tardis-sn/carsus/pull/313/commits/0da5cbbb0d0cdd085b829b36608b5887860bb2e2).